.flowconfig

```diff
@@ -0,0 +1,9 @@
+[ignore]
```

Comment:
I didn't see anything in the [docs](https://flow.org/en/docs/cli/) about providing a path for a config file.